Packs/Microsoft365Defender/Layouts/layoutscontainer-Microsoft_365_Defender_-_Layout.json

```diff
@@ -0,0 +1,410 @@
+                                "index": 2,
+                                "sectionItemType": "field",
+                                "startCol": 0
+                            },
+                            {
+                                "endCol": 2,
+                                "fieldId": "eventtype",
+                                "height": 22,
+                                "id": "5f8b8cd0-a5ca-11eb-9ef8-93e7f36c667d",
+                                "index": 3,
+                                "sectionItemType": "field",
+                                "startCol": 0
+                            },
+                            {
+                                "dropEffect": "move",
+                                "endCol": 2,
+                                "fieldId": "microsoft365defenderstatus",
+                                "height": 22,
+                                "id": "65dfba70-a5ca-11eb-9ef8-93e7f36c667d",
+                                "index": 4,
+                                "listId": "caseinfoid-383ed1f0-a5ca-11eb-9ef8-93e7f36c667d",
+                                "sectionItemType": "field",
+                                "startCol": 0
+                            },
+                            {
+                                "endCol": 2,
+                                "fieldId": "severity",
+                                "height": 22,
+                                "id": "8c7f6c70-a5ca-11eb-9ef8-93e7f36c667d",
+                                "index": 5,
+                                "sectionItemType": "field",
+                                "startCol": 0
+                            },
+                            {
+                                "dropEffect": "move",
+                                "endCol": 2,
+                                "fieldId": "microsoft365defendertags",
+                                "height": 22,
+                                "id": "9235fa80-a5ca-11eb-9ef8-93e7f36c667d",
+                                "index": 6,
+                                "listId": "caseinfoid-383ed1f0-a5ca-11eb-9ef8-93e7f36c667d",
+                                "sectionItemType": "field",
+                                "startCol": 0
+                            }
+                        ],
+                        "maxW": 3,
+                        "minH": 1,
+                        "minW": 1,
+                        "moved": false,
+                        "name": "Microsoft 365 Defender Incident Data",
+                        "static": false,
+                        "w": 3,
+                        "x": 0,
+                        "y": 0
+                    }
+                ],
+                "type": "custom"
+            }
+        ]
+    },
+    "group": "incident",
+    "id": "Microsoft 365 Defender - Layout",
+    "name": "Microsoft 365 Defender - Layout",
+    "system": false,
+    "version": -1,
+    "fromVersion": "6.0.0",
```

Comment:
this layout is for 6.0.0 you need to add an old_layout for 5.5.0-6.0.0 - you can do this by starting a self-service with 5.5.0 and importing your content and creating layout there in the same manner done in your local env.

Reply:
Done